src/libraries/System.Globalization/System.Globalization.sln

```diff
@@ -33,6 +33,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ref", "ref", "{A93AFF96-DB2
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "gen", "gen", "{0378EF1C-9838-4AD0-867D-506FB02F8BBB}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hybrid.Tests", "tests\Hybrid\Hybrid.Tests.csproj", "{4134CD4D-1E8F-45D4-883C-B08A1DB5B274}"
+EndProject
```

Comment:
There is already https://github.com/dotnet/runtime/tree/main/src/libraries/System.Globalization/tests/Hybrid created. I'm not quite sure why it shows here as added project.

Reply:
I don't see this project added to the solution file: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Globalization/System.Globalization.sln

Reply:
Thank you, after reading the whole review I have the following idea:
I will rename `Hybrid.Tests.csproj` to indicate clearly it's for ios and add a similar one in the same location for wasm. I will apply corresponding changes to the solution because it seems some features might get broken when solution does not have correct references to the lib tests, see: https://github.com/dotnet/runtime/pull/34656.
How does it sound?

Reply:
Thanks for adding to sln,  I forgot that in my PR. This looks good to me.